src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -63,6 +63,12 @@ public class HostStatus
         [JsonProperty(PropertyName = "processUptime", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public long ProcessUptime { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the filesystem is read-only
```

Comment:
wondering if this should be enum (enabled/disabled/unknown)

basically we return enabled/disabled if we are sure and unknown otherwise. this will allow clients differentiate between editable = false since app is actually not editable / we failed to determine the status for some reason / scope this change to specific skus and let clients use some fallback logic if this api returns unknown.

Reply:
Not sure how that adds more value to the clients, since we want to be very restrictive with this feature, i.e. _only_ make the app contents editable if the host returns True here. Otherwise, both for Unknown/False, we want to turn it off.

Reply:
Discussed offline. Will be taking this approach since it accounts better for other SKUs (for example, we will return Unknown for non-consumption SKUs until we figure out the check for them).